syntax.json

```
@@ -397,5 +397,21 @@
         }
       }
     ]
+  },
+  {
+    "language": " TeX",
+    "markers": [
+      {
+        "type": "line",
+        "pattern": "%"
+      },
+      {
+        "type": "block",
+        "pattern": {
+          "start": "\begin{comment}",
+          "end": "\end{comment}"
```

```suggestion
          "start": "\\begin{comment}",
          "end": "\\end{comment}"
```
escape them like this?